src/App.js

```diff
@@ -1,15 +1,42 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from "react-redux";
+import { selectorListFilms, selectorGenre } from "./modules/selectors/index";
```

Comment:
basically we don't need to write `/index` at the end of the import path
`import { selectorListFilms, selectorGenre } from "./modules/selectors` should be enough